src/coreclr-debug.ts

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import * as child_process from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+
+var _coreClrDebugDir: string;
+var _debugAdapterDir: string;
+var _channel: vscode.OutputChannel;
```

Comment:
I'm fairly new to node/js. Do I need to use let throughout or just for these file level variables? My understanding is that let has block scope semantics vs global scope semantics?